File: BE/Boek.Infrastructure/ViewModels/Campaigns/Mobile/Customers/CustomerCampaignMobileViewModel.cs
namespace Boek.Infrastructure.ViewModels.Campaigns.Mobile.Customers
{
    public class CustomerCampaignMobileViewModel
    {
        public List<HierarchicalCustomerCampaignMobileViewModel> hierarchicalCustomerCampaigns { get; set; }
        public List<UnhierarchicalCustomerCampaignMobileViewModel> unhierarchicalCustomerCampaigns { get; set; }
    }
}

File: Mobile/objects/viewmodels/Campaigns/Mobile/CustomerCampaignMobileViewModel.ts
import { HierarchicalCustomerCampaignMobileViewModel } from "./HierarchicalCustomerCampaignMobileViewModel";
import { UnhierarchicalCustomerCampaignMobileViewModel } from "./UnhierarchicalCustomerCampaignMobileViewModel";

export interface CustomerCampaignMobileViewModel {
    hierarchicalCustomerCampaigns: HierarchicalCustomerCampaignMobileViewModel[];
    unhierarchicalCustomerCampaigns: UnhierarchicalCustomerCampaignMobileViewModel[];
}